source/includes/fundamentals/code-snippets/LongCodec.java

```diff
@@ -0,0 +1,29 @@
+package fundamentals;
```

Comment:
Note to tech reviewer: I'm a little uncertain about this example code. The [example provided in the ticket](https://github.com/mongodb/mongo-java-driver/blob/master/bson/src/main/org/bson/codecs/StringCodec.java) didn't convert between Long and Datetime, so I wrote my own example. Let me know if this needs to be updated!